src/ts/translations/de.ts

```diff
@@ -0,0 +1,41 @@
+import {Translation} from './translation-types';
+
+let lang: Translation = {
+  language: 'german',
+  settings: {
+    panel: {
+      audioQuality: 'Audioqualität',
+      videoQuality: 'Videoqualität',
+      speed: 'Geschwindigkeit',
+      audioTrack: 'Audiospur',
+    },
+    fonts: {
+      fontSize: 'Größe',
+      fontFamily: 'Schriftart',
+      fontColor: 'Farbe',
+      fontOpacity: 'Deckkraft',
+      characterEdge: 'Ränder',
+      backgroundColor: 'Hintergrundfarbe (Text)',
+      backgroundOpacity: 'Hintergrunddeckkraft (Text)',
+      windowColor: 'Hintergrundfarbe (Textbox)',
+      windowOpacity: 'Hintergrunddeckkraft (Textbox)',
+    },
+  },
+  labels: {
+    subtitles: 'Untertitel',
+    subtitleSettings: 'Subtitles settings',
+    reset: 'Zurücksetzen',
+    back: 'Zurück',
+    normal: 'Normal',
+  },
+  messages: {
+    remainingTime: 'Ad: {remainingTime} secs',
+    connectingTo: function(castDeviceName: string): string {
```

Comment:
This and the next should go into a `cast` section

Reply:
I like this function-based pattern but I think the downside is that this prohibits us from supplying `Translation` objects as JSON. If they were pure JSON objects then they could much easier be used with translation software. The object property hierarchy could easily be converted into i18next-type strings and back if necessary, e.g. `en.settings.audioQuality` <=> `'en.settings.audioQuality'`.

So in the light of that, it might make more sense to keep a `Translation` object function free and provide a separate function on the `ì18n` object that must be used where placeholders are desired.